configs/rbe/gen.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+#
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# The script is used to generate rbe configurations.
+#
+
+set -eu
+
+cd $(dirname $BASH_SOURCE)/../..
+BASEDIR=configs/rbe
+
+BAZEL_VERSION=4.1.0
```

Comment:
What is this version number used for? The configs checked into the `master` branch should probably be built using the latest Bazel rolling release instead of the latest LTS version.

Reply:
The tool will pass this number to `bazelisk` when auto-detecting toolchains in the container. I think the version should match which we use on Bazel CI.

Using rolling release will not re-generate the configuration unless we manually re-run the tool again.